src/release/publisher.ts

```diff
@@ -1190,6 +1246,28 @@ export function isAwsCodeArtifactRegistry(registryUrl: string | undefined) {
   return registryUrl && AWS_CODEARTIFACT_REGISTRY_REGEX.test(registryUrl);
 }
 
+/**
+ * Info extracted from AWS CodeArtifact URL
+ */
+export interface AwsCodeArtifactInfo {
```

Comment:
No need for this to be exported and part of the public API.

```suggestion
interface AwsCodeArtifactInfo {
```